### ISSUE TITLE
Stop Running Command

### ISSUE THREAD

**Issue by mmnoureldin** [BishopFox/sliver#940]
Is there any way to stop a running command such as the ones running from execute-assembly, the only way I can see is using ctrl-c, but it stops sliver  itself.

This behavior can be see when using aliases too such as "Rubeus monitor" command which runs forever until stopped manually.

**Comment by moloch--** [comment]
Currently no, not really any way to stop an individual task/command after it's been issued aside from using `--timeout`

**Comment by rkervella** [comment]
And even using `--timeout`, you won't really stop the command, you'll just gain control in the client again upon timeout.

**Comment by rkervella** [comment]
A workaround for things like `Rubeus monitor` would be to run two different clients, and setting a really long timeout on the one running the `monitor` command.

**Comment by mmnoureldin** [comment]
> A workaround for things like `Rubeus monitor` would be to run two different clients, and setting a really long timeout on the one running the `monitor` command.

Yup, I already thought of that, but in general these little changes will make much improvements in user experience.

**Comment by rkervella** [comment]
> Yup, I already thought of that, but in general **these little changes** will make much improvements in user experience.

Here's the thing though, it is not "a little change".

**Comment by moloch--** [comment]
Yea, implementing this would be a significant undertaking, it may seem simple from a user perspective but it's not from a developer standpoint.

**Comment by n3rada** [comment]
Hi there,

Any "new" way to handle the rubeus monitoring thing?
